wsagent/che-core-api-git-shared/src/main/java/org/eclipse/che/api/git/shared/LogRequest.java

```diff
@@ -21,21 +20,96 @@
  */
 @DTO
 public interface LogRequest extends GitRequest {
-    /** Filter revisions list by range of files. */
+
+    /** @return revision range since */
+    String getRevisionRangeSince();
+
+    /** @set revision range since */
+    void setRevisionRangeSince(String revisionRangeSince);
+
+    /**
+     * Create a LogRequest object based on a given revision range since.
+     *
+     * @param revisionRangeSince
+     *         revision range since
+     * @return a LogRequest object
+     */
+    LogRequest withRevisionRangeSince(String revisionRangeSince);
+
+    /** @return revision range until */
+    String getRevisionRangeUntil();
+
+    /** @set revision range until */
+    void setRevisionRangeUntil(String revisionRangeUntil);
+
+    /**
+     * Create a LogRequest object based on a given revision range until.
+     *
+     * @param revisionRangeUntil
+     *         revision range until
+     * @return a LogRequest object
+     */
+    LogRequest withRevisionRangeUntil(String revisionRangeUntil);
+
+    /** @return the integer value of the number of commits that will be skipped when calling log API */
```

Comment:
the integer value of the number of commits that will be skipped when calling log command ?